README.md

```diff
@@ -1,45 +1,80 @@
 Introduction
 ============
 
-With the prevalence of multi-CPU computers and multi-core CPUs, the opportunity exists to speed-up the execution of applications in general and algorithms that lend themselves to parallelisation specifically.
```

Comment:
I may have cut the width down 😬 I hope that's ok.

Reply:
No problem (handy when using _vi_ as editor)